packages/argo-run/src/dev.ts

```diff
@@ -73,6 +88,17 @@ export async function dev(...args: string[]) {
 
   log(`Your extension is available at ${fileUrl}`);
 
+  // we could replace this with a check for extension type
+  // to only show the query parameters for post purchase extensions
+  if (extensionConfig) {
```

Comment:
In the future, we could also listen for file changes it update the config so consumers would not have to restart upon config change, but I think for now that's fine